app/controllers/iiif_controller.rb

```diff
@@ -71,7 +71,7 @@ def set_image_response_headers
   end
 
   def set_attachment_content_disposition_header
-    response.headers['Content-Disposition'] = "attachment;filename=#{params[:identifier]}.#{params[:format]}"
+    response.headers['Content-Disposition'] = "attachment;filename=#{identifier_params[:file_name]}.#{params[:format]}"
```

Comment:
No, definitely not - filenames do not usually include the druid.  The goal is to have the downloaded filename be just the filename as it actually exists in the object and without the pre-prepended object identifier (currently DRUID/).   This was the original behavior before IIIF was implemented.  I have confirmed with Rob and Stu that it is acceptable and within IIIF spec to go back to this behavior.  The discussion is about the best way to re-implement this.